modules/ui/src/main/scala/lucuma/ui/syntax/time.scala

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.ui.syntax
+
+import cats.syntax.eq.*
+import lucuma.core.util.TimeSpan
+
+trait time:
+  extension (timespan: TimeSpan)
+    /**
+     * Format a timespan in the format `${hh}hrs ${mm}mins`
+     */
+    def toHoursMinutes: String =
```

Comment:
Colud this be a `Forrmat`?

Reply:
Could be. But that does need a reverse `getOption` which means we'd have to parse the output and I'm not sure that's worth it

Reply:
👍